shared/translations/languages/de.json

```diff
@@ -803,7 +803,11 @@
     "Enter passcode to continue": "Passwort eingeben, um fortzufahren",
     "Please enter passcode to securely sign your transaction": "Gib bitte das Passwort ein, um die Transaktion freizugeben",
     "Sign to verify access": "Freigeben, um den Zugriff zu bestätigen",
-    "Success!": "Erfolgreich!"
+    "Success!": "Erfolgreich!",
+    "It may take a few seconds to verify": "It may take a few seconds to verify",
+    "Passcode verified!": "Passcode verified!",
+    "Provide your passcode to view recovery words.": "Provide your passcode to view recovery words.",
```

Comment:
Do we already have translations for these?

Reply:
No we don't. Making a list and will request from translation